debian/package.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+
+set -euo pipefail
+
+PROGRAM="$(basename "$0")"
+
+VERBOSE=false
+
+REZOLUS=/mnt/rezolus
+OUTPUT=/mnt/output
+RELEASE=0
+CHOWN=
+
+help() {
+    cat <<EOF
+$PROGRAM - Build rezolus debian packages.
+
+USAGE:
+    $PROGRAM <FLAGS>
+
+OPTIONS:
+    -h|--help       Show this help text.
+    -v|--verbose    Display the commands run by this script.
+
+    --release       The release number to use for the package. [default: $RELEASE]
+    --rezolus-dir   The directory that the rezolus source is stored in. [default: $REZOLUS]
+    --output-dir    The directory to place the output artifacts in. [default: $OUTPUT]
+    --chown         Change the ownership of the resulting package files.
+
+USE WITH DOCKER:
+    This script is intended to be run within a debian-based docker container.
+    As an example, consider building for ubuntu focal:
+
+    docker run -it --rm \\
+        -v \$(pwd):/mnt/rezolus \\
+        -v \$(pwd)/target/debian:/mnt/output \\
+        ubuntu:focal /mnt/rezolus/debian/package.sh --release 0 --chown \$(id -u) --verbose
```

Comment:
I'd rather that we use a supported distro/release here. Consider using `noble` as the example